### ISSUE TITLE
Logistic Regression Classifier 

### ISSUE THREAD

**Issue by zknus** [zyzisastudyreallyhardguy/Graph-Group-Discrimination#7]
Hi, Thanks for your great work!

I have a question regarding the implementation of the test classifier in your code. In your paper, you mentioned that the classifier would be a logistic regression model, but in the code, a linear fully-connected (FC) classifier is used instead. I was wondering if you could provide some insights into this implementation. Thanks!

https://github.com/zyzisastudyreallyhardguy/Graph-Group-Discrimination/blob/7cf72db392a758f041031c24207c747a496a5683/GGD_ogbn_arxiv_1epoch/ggd.py#L74-L78

**Comment by zyzisastudyreallyhardguy** [comment]
Hi zknus,

This is a common practice that can be found in many well-known graph self-supervised learning baselines, e.g., DGI, MVGRL, GMI, etc.

In my opinion, people use this setting because logistic regression can be thought of as a single linear layer followed by a sigmoid activation function. For multi-class classification problems, the softmax function is used instead of the sigmoid function, which is equivalent to a single linear layer followed by a softmax activation in a neural network.

Hope this addresses your concern.

Warmest Regards,
YIZHEN ZHENG

**Comment by zknus** [comment]
Hi Yizhen,

Thanks very much!
